.travis.yml

```diff
@@ -8,10 +8,10 @@ matrix:
   include:
     - os: linux
       compiler: clang
-      env: BUILD_TYPE=default CFLAGS='-fsanitize=address -fsanitize=undefined'
+      env: BUILD_TYPE=default CFLAGS='-fsanitize=address -fsanitize=undefined -Werror'
     - os: linux
       compiler: clang-5.0
-      env: BUILD_TYPE=default CFLAGS='-fsanitize=address -fsanitize=undefined'
+      env: BUILD_TYPE=default CFLAGS=' -fsanitize=address -fsanitize=undefined -Werror'
```

Comment:
It seems like a space snuck in just before `-fsanitize`.

Reply:
Nice catch!